src/Uno.UWP/Storage/FileIO.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Globalization;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno.Extensions;
+using Windows.Foundation;
+using Windows.Storage.FileProperties;
+using Windows.Storage.Streams;
+
+namespace Windows.Storage
+{
+	public partial class FileIO
+	{
+
+		private static async Task AppendWriteTextAsync(global::Windows.Storage.IStorageFile file, string contents, global::Windows.Storage.Streams.UnicodeEncoding encoding, bool append)
+		{
+			if (file is null)
+			{
+				throw new NullReferenceException("StorageFile cannot be null");
+			}
+
+			global::System.Text.Encoding encodingForWriter = global::System.Text.Encoding.UTF8;
+			switch (encoding)
+			{
+				case global::Windows.Storage.Streams.UnicodeEncoding.Utf8:
+					encodingForWriter = global::System.Text.Encoding.UTF8;
+					break;
+				case global::Windows.Storage.Streams.UnicodeEncoding.Utf16LE:
+					encodingForWriter = global::System.Text.Encoding.Unicode;
+					break;
+				case global::Windows.Storage.Streams.UnicodeEncoding.Utf16BE:
+					encodingForWriter = global::System.Text.Encoding.BigEndianUnicode;
+					break;
+			}
+
+			using (Stream fileStream = await file.OpenStreamForWriteAsync())
```

Comment:
[`File.Open`](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.open?view=netframework-4.8#System_IO_File_Open_System_String_System_IO_FileMode_System_IO_FileAccess_) has an append mode, no need to seek, same comment for the other method below.

Reply:
OpenStreamForWriteAsync has no mode (append/rewrite) parameter, also StreamWriter with Stream parameter has no such parametr. Or, at least, I cannot find it.

Reply:
You can [create a `StreamWriter` from a `File`](https://docs.microsoft.com/en-us/dotnet/api/system.io.streamwriter.-ctor?view=netframework-4.8#System_IO_StreamWriter__ctor_System_IO_Stream_System_Text_Encoding_).

Reply:
But I have Windows.Storage.StorageFile , not File ?